dali/pipeline/operator/sequence_shape_test.cc

```diff
@@ -384,7 +386,8 @@ class SequenceShapeBroadcastTest<TensorList<Backend>>
       batch.SetLayout(layout);
     }
     this->template FillBatch<T>(batch);
-    return {std::move(batch), expanded_like(batch)};
+    auto expanded = expanded_like(batch);
```

Comment:
what was the issue here?

Reply:
The call to `expand_like` could have happened after `std::move(batch)` for the first item in the result. It's UB.